cmd/juju/caas/add.go

```diff
@@ -595,6 +595,10 @@ func (c *AddCAASCommand) validateCloudRegion(ctx *cmd.Context, cloudRegion strin
 				}
 				return details.CloudType, nil
 			}
+			if region == "" && details.DefaultRegion != "" {
```

Comment:
For safety we should still check that details.DefaultRegion is in the details.RegionsMap
ie if region is "", set it to default region can continue as before